standalone-metastore/metastore-server/src/main/sql/mssql/hive-schema-4.0.0.mssql.sql

```diff
@@ -424,7 +424,10 @@ CREATE INDEX MV_UNIQUE_TABLE ON MV_CREATION_METADATA (TBL_NAME,DB_NAME);
 CREATE TABLE MV_TABLES_USED
 (
     MV_CREATION_METADATA_ID bigint NOT NULL,
-    TBL_ID bigint NOT NULL
+    TBL_ID bigint NOT NULL,
+    INSERTED_COUNT bigint NOT NULL DEFAULT 0,
+    UPDATED_COUNT bigint NOT NULL DEFAULT 0,
+    DELETED_COUNT bigint NOT NULL DEFAULT 0
```

Comment:
primary key seems to be missing

Reply:
Added pk definitions to schema and upgrade sqls.